src/status_im/multiaccounts/login/core.cljs

```diff
@@ -521,6 +521,7 @@
                                 (assoc-in [:multiaccount :multiaccounts/first-account] first-account?))
                ::get-tokens [network-id accounts recovered-account?]}
               (finish-keycard-setup)
+              (initialize-appearance)
```

Comment:
how this related to PR ?

Reply:
Hi @flexsurfer, Thank you very much for reviewing PR.

So an account's default theme on creation is the system theme.
But while creating the account we didn't initialize the theme till now, we kept whatever theme was on onboarding.
So, until now onboarding was also using a system theme, so it was ok.
But now onboarding uses a dark theme, so we need to initialize that for creating an account too.